docker/lookaside.go

```diff
@@ -45,11 +52,11 @@ type registryNamespace struct {
 }
 
 // signatureStorageBase is an "opaque" type representing a lookaside Docker signature storage.
-// Users outside of this file should use configuredSignatureStorageBase and signatureStorageURL below.
+// Users outside of this file should use ConfiguredSignatureStorageBase and signatureStorageURL below.
 type signatureStorageBase *url.URL // The only documented value is nil, meaning storage is not supported.
 
-// configuredSignatureStorageBase reads configuration to find an appropriate signature storage URL for ref, for write access if “write”.
-func configuredSignatureStorageBase(sys *types.SystemContext, ref dockerReference, write bool) (signatureStorageBase, error) {
+// ConfiguredSignatureStorageBase reads configuration to find an appropriate signature storage URL for ref, for write access if “write”.
+func ConfiguredSignatureStorageBase(sys *types.SystemContext, ref dockerReference, write bool) (signatureStorageBase, error) {
```

Comment:
… maybe name this something like `SignatureStorageBaseURL` (`RegistriesDSignatureStorageBaseURL`? `RegistriesDSigstoreBaseURL`?) and have it return an URL directly?

The documentation should also link to the “docker/distribution registries—separate storage” section of `docs/signature-protocols.md` for explanation of what a “base URL” is, and explicitly say something like
> Warning:  This function only exposes configuration in registries.d; just because this function returns an URL does not mean that the URL will be used by c/image/docker (e.g. if the registry natively supports X-R-S-S).